ironfish-rust-nodejs/src/structs/mod.rs

```diff
@@ -22,3 +22,6 @@ pub use witness::*;
 
 mod key_packages;
 pub use key_packages::*;
+
+mod unsigned;
+pub use unsigned::*;
```

Comment:
should we define a new module for UnsignedTransaction or use the existing transaction module?

Reply:
Yea probably should since that is the current status quo, although I would like to refactor to make them submodules